src/site/content/en/blog/state-of-css2021/index.md

```diff
@@ -0,0 +1,52 @@
+If you haven't used CSS Grid, now is a great time to [learn it](/learn/css/grid/).
+
+As more developers use grid, [awareness of subgrid]([https://2021.stateofcss.com/en-US/features/layout/#subgrid](https://2021.stateofcss.com/en-US/features/layout/#subgrid)) is also growing. Subgrid is already available in Firefox and will land in Chrome as part of the work by the Microsoft Team on [GridNG](https://blogs.windows.com/msedgedev/2021/08/10/compat2021-css-grid-gridng/).
+
+Other features with strong growth in usage and awareness are 
+[`aspect-ratio`]([https://2021.stateofcss.com/en-US/features/layout/#aspect_ratio](https://2021.stateofcss.com/en-US/features/layout/#aspect_ratio)`), [scroll snap]([https://2021.stateofcss.com/en-US/features/interactions/#scroll_snap](https://2021.stateofcss.com/en-US/features/interactions/#scroll_snap)), and [custom properties]([https://2021.stateofcss.com/en-US/features/other-features/#variables](https://2021.stateofcss.com/en-US/features/other-features/#variables)).
+
+# Browser compatibility
+
+Browser compatibility is a common pain point for web developers and, to learn more about what CSS features cause the most issues, the survey asked, "Are there any CSS features you have difficulties using because of differences between browsers?"
+
+{% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/rjslQmdTEZr8lNz0EbHC.png", alt="Survey results for features with differences between browsers. The top three responses are grid, flexbox gap, and subgrid.", width="800", height="909" %} 
+
+Survey results like these help guide prioritization for browser vendors. Many of the features are part of the [Compat 2021 effort](/compat2021-holiday-update/) based on the [MDN Browser Compatibility Report 2020](https://insights.developer.mozilla.org/reports/mdn-browser-compatibility-report-2020.html), and others like subgrid are now being [proposed](https://github.com/web-platform-tests/interop-2022/issues/1) as focus areas for Interop 2022.
+
+# What's missing in CSS?
+
+The survey also asked, "Which feature would you most like to be able to use in CSS today?"
+
+{% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/cZesmseagvBYouHVaRJO.png", alt="Survey results for the question what's missing in CSS. The top three responses are container queries, parent selector, and browser support.", width="800", height="617" %}
+
+Container Queries was the overall "winner", matching the [2020 results](https://2020.stateofcss.com/en-US/opinions/#currently_missing_from_css). Container queries are part of the [new responsive](/new-responsive/). Chrome is currently working on an experimental implementation, and funding the work of Miriam Suzanne, as she develops the specification in the CSS Working Group. You can try it out by going to `about:flags`, searching for, and enabling, the **Container Queries** flag.
```

Comment:
```suggestion
Container Queries was the overall "winner", matching the [2020 results](https://2020.stateofcss.com/en-US/opinions/#currently_missing_from_css). Container queries enable [component-based responsive](/new-responsive/) design by allowing elements to query their parent selector for style changes. This is a much more local, specified scope than is currently possible with media queries. 

Chrome is currently working on an experimental implementation, and funding the work of Miriam Suzanne, as she develops the specification in the CSS Working Group. You can try it out by going to `about:flags`, searching for, and enabling, the **Container Queries** flag. You can learn more about container queries, as well as see some demos that use them in the latest series of [Designing in the Browser](https://www.youtube.com/watch?v=gCNMyYr7F6w&list=PLNYkxOF6rcIDI0QtJvW6vKonTxn6azCsD&index=14).
```

Reply:
Also suggestion to embed this video under this section, since it does an overview:

https://www.youtube.com/watch?v=gCNMyYr7F6w&list=PLNYkxOF6rcIDI0QtJvW6vKonTxn6azCsD&index=14